src/extension/auth.js

```diff
@@ -49,9 +51,29 @@ export async function addAuthTokenHeaders() {
           requestMethods: ['get', 'post', 'delete'],
           resourceTypes: ['xmlhttprequest'],
         },
-      });
-      id += 1;
-      log.debug(`addAuthTokensHeaders: added admin auth header rule for ${owner}`);
+      };
+
+      const corsRule = {
+        id: id++,
+        priority: 1,
+        action: {
+          type: 'modifyHeaders',
+          responseHeaders: [{
+            header: 'Access-Control-Allow-Origin',
+            operation: 'set',
+            value: '*',
```

Comment:
Shouldn't this be set to `tools.aem.live` or `labs.aem.live` respectively?

Reply:
How would I know which one to set though? This is set at the time of login (on their aem.page/live or host).

Think I should create two rules then?

Reply:
I think also `initiatorDomains` will have the same effect as being explicit with the ACAO header.